runtime/v2/manager.go

```diff
@@ -155,6 +156,10 @@ func (m *TaskManager) Create(ctx context.Context, id string, opts runtime.Create
 			defer cancel()
 			_, errShim := shim.Delete(dctx)
 			if errShim != nil {
+				if errors.As(errShim, &context.DeadlineExceeded) {
```

Comment:
It is incorrect to alter `context.DeadlineExceeded`. You perhaps meant `errors.Is`?
https://golang.org/pkg/errors/#Is

Reply:
Thanks. updated.